database/seeders/ModelSeeders/CountrySeeder.php

```diff
@@ -270,7 +270,7 @@ private function getCountryData()
             ['acronym' => 'TM', 'name' => 'Turkmenistan', 'display' => 0],
             ['acronym' => 'TN', 'name' => 'Tunisia', 'display' => 1],
             ['acronym' => 'TO', 'name' => 'Tonga', 'display' => 0],
-            ['acronym' => 'TR', 'name' => 'Turkey', 'display' => 1],
+            ['acronym' => 'TR', 'name' => 'Türkiye', 'display' => 1],
```

Comment:
rest of this table has diacritics removed, idk if it matters but may as well be consistent.

ex. Saint Barthélemy -> Saint Barthelemy, Åland Islands -> Aland Islands